services/mailer/mailer.go

```diff
@@ -210,8 +210,14 @@ func (s *smtpSender) Send(from string, to []string, msg io.WriterTo) error {
 		}
 	}
 
-	if err = client.Mail(from); err != nil {
-		return fmt.Errorf("Mail: %v", err)
+	if opts.OverrideEnvelopeFrom {
+		if err = client.Mail(opts.EnvelopeFrom); err != nil {
+			return fmt.Errorf("Mail: %v", err)
+		}
+	} else {
+		if err = client.Mail(from); err != nil {
+			return fmt.Errorf("Mail: %v", err)
```

Comment:
Just remember that Go spec prefer the error strings starting with lower-case 😂

LGTM.